tools/system_libs.py

```diff
@@ -381,11 +313,15 @@ def build_objects(self, build_dir):
         cmd.remove('-g')
       else:
         cmd += cflags
+      cmd = self.customize_build_cmd(cmd, src)
       commands.append(cmd + ['-c', src, '-o', o])
       objects.append(o)
     run_build_commands(commands)
     return objects
 
+  def customize_build_cmd(self, cmd, filename):  # noqa
```

Comment:
Maybe add a comment for this new hook? Explaining the params would be good in particular.